src/module/combat/combat.js

```diff
@@ -552,6 +552,153 @@ export class CombatSFRPG extends Combat {
         return this.getIndexOfFirstUndefeatedCombatant() === null;
     }
 
+    average(array) {
+        let total = 0;
+        let count = 0;
+        jQuery.each(array, function(index, value) {
+            total += value;
+            count++;
+        });
+        return total / count;
+    }
+
+    getEncounterInfo() {
+        if (!this.flags.sfrpg) {
+            this.flags.sfrpg = [];
+        }
+
+        const playerInfo = this.calculateAPL();
```

Comment:
We can make these array indices look a bit more ergonomic by destructuring them
```js
const [PCs, APL] = this.calculateAPL();
this.flags.sfrpg.PCs = PCs;
this.flags.sfrpg.APL = APL;
```
for example

Reply:
This makes sense, I didn't know you could assign like this in JS.

Would it make more sense to just directly assign to the flags, or would this be bad practice?

```suggestion
        [this.flags.sfrpg.PCs, this.flags.sfrpg.APL] = this.calculateAPL();
```

Reply:
I'm not sure if that even works tbh, but even if it does I prefer `[PCs, APL]` be sure it provides more information about what is coming out of the function